distribution/lib/Standard/Table/0.0.0-dev/src/Data/Table.enso

```diff
@@ -726,6 +729,85 @@ type Table
         result = Table.new new_columns
         on_problems.attach_problems_after result problem_builder.to_vector
 
+    ## Replaces the first, last, or all occurrences of term with new_text in each text row of selected columns.
+       If `term` is empty, the function returns the table unchanged.
+
+       This method follows the exact replacement semantics of the `Text.replace` method.
+
+       Arguments:
+       - columns: The columns of the table in which to perform text replacement.
```

Comment:
This suggests it only accepts column references, but it actually accepts any Column_Selectors.

So I'd suggest something along the lines of:
```suggestion
       - columns: Column selection criteria or a column name or index.
```
like in other similar functions.